reconcile/utils/ocm.py

```diff
@@ -823,13 +830,43 @@ def _init_addons(self):
         api = f"{CS_API_BASE}/v1/addons"
         self.addons = self._get_json(api).get("items")
 
+    def _init_version_gates(self):
+        """Returns a list of version gates"""
+        api = f"{CS_API_BASE}/v1/version_gates"
+        self.version_gates = self._get_json(api).get("items")
+
     def get_addon(self, id):
         for addon in self.addons:
             addon_id = addon["id"]
             if id == addon_id:
                 return addon
         return None
 
+    def get_version_gates(self, version_prefix: str) -> list[dict[str, Any]]:
+        if not self.init_version_gates:
+            self._init_version_gates()
+        return_gates = []
+        for gate in self.version_gates:
+            if gate["version_raw_id_prefix"] == version_prefix:
+                return_gates.append(gate)
```

Comment:
A comprehension would be more pythonic
```suggestion
        return_gates = [g for g in self.version_gates if g['"version_raw_id_prefix"] == version_prefix]
```